teamcity/resource_build_trigger_schedule.go

```diff
@@ -65,6 +66,44 @@ func resourceBuildTriggerSchedule() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Optional: true,
 			},
+			"cron_schedule": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Computed: true,
```

Comment:
What's the reason for making this computed?

Reply:
Removed.